repo2docker/buildpacks/r.py

```diff
@@ -295,33 +327,30 @@ def get_build_scripts(self):
                 """,
             ),
             (
-                "${NB_USER}",
-                # Install a pinned version of IRKernel and set it up for use!
+                "root",
+                # RStudio's CRAN mirror needs this to figure out which binary package to serve.
+                # If not set properly, it will just serve up source packages
+                # Quite hilarious, IMO.
+                # See https://docs.rstudio.com/rspm/admin/binaries.html
```

Comment:
https://docs.rstudio.com/rspm/admin/binaries.html is a 404 at the moment :disappointed:

Reply:
Found a replacement link!